docs/openfaas-cloud/secrets.md

```diff
@@ -0,0 +1,45 @@
+OpenFaaS Cloud **secret** objects lets you store and manage sensitive information securely
```

Comment:
Why do we have secret in bold? There's not a thing called a secret object

Reply:
👍